lib/datadog/tracing/contrib/mysql2/ext.rb

```diff
@@ -15,6 +15,8 @@ module Ext
           TAG_DB_NAME = 'mysql2.db.name'.freeze
           TAG_COMPONENT = 'mysql2'.freeze
           TAG_OPERATION_QUERY = 'query'.freeze
+
+          DB_SYSTEM = 'mysql'.freeze
```

Comment:
I suggest moving this to `contrib/ext.rb`, so it sits near all other `db.system` constants.

Reply:
Actually, it's fine here, nvm.